pysat/_meta.py

```diff
@@ -119,294 +79,200 @@ class Meta(object):
-        meta['name2'] = {'units':string}
+        meta['name2'] = {'units': string}
         # units are set to '' by default
-        meta['name3'] = {'long_name':string}
+        meta['name3'] = {'long_name': string}
 
         # assigning custom meta parameters
-        meta['name4'] = {'units':string, 'long_name':string
-                         'custom1':string, 'custom2':value}
-        meta['name5'] = {'custom1':string, 'custom3':value}
+        meta['name4'] = {'units': string, 'long_name': string
+                         'custom1': string, 'custom2': value}
+        meta['name5'] = {'custom1': string, 'custom3': value}
 
         # assign multiple variables at once
-        meta[['name1', 'name2']] = {'long_name':[string1, string2],
-                                    'units':[string1, string2],
-                                    'custom10':[string1, string2]}
+        meta[['name1', 'name2']] = {'long_name': [string1, string2],
+                                    'units': [string1, string2],
+                                    'custom10': [string1, string2]}
 
         # assiging metadata for n-Dimensional variables
         meta2 = pysat.Meta()
-        meta2['name41'] = {'long_name':string, 'units':string}
-        meta2['name42'] = {'long_name':string, 'units':string}
-        meta['name4'] = {'meta':meta2}
+        meta2['name41'] = {'long_name': string, 'units': string}
+        meta2['name42'] = {'long_name': string, 'units': string}
+        meta['name4'] = {'meta': meta2}
+
         # or
         meta['name4'] = meta2
         meta['name4'].children['name41']
 
         # mixture of 1D and higher dimensional data
         meta = pysat.Meta()
-        meta['dm'] = {'units':'hey', 'long_name':'boo'}
-        meta['rpa'] = {'units':'crazy', 'long_name':'boo_whoo'}
+        meta['dm'] = {'units': 'hey', 'long_name': 'boo'}
+        meta['rpa'] = {'units': 'crazy', 'long_name': 'boo_whoo'}
         meta2 = pysat.Meta()
-        meta2[['higher', 'lower']] = {'meta':[meta, None],
-                                      'units':[None, 'boo'],
-                                      'long_name':[None, 'boohoo']}
+        meta2[['higher', 'lower']] = {'meta': [meta, None],
+                                      'units': [None, 'boo'],
+                                      'long_name': [None, 'boohoo']}
 
         # assign from another Meta object
         meta[key1] = meta2[key2]
 
         # access fill info for a variable, presuming default label
         meta[key1, 'fill']
+
         # access same info, even if 'fill' not used to label fill values
         meta[key1, meta.fill_label]
 
 
         # change a label used by Meta object
         # note that all instances of fill_label
-        # within the meta object are updated
-        meta.fill_label = '_FillValue'
-        meta.plot_label = 'Special Plot Variable'
+        # within the meta object are NOT updated FIX??
```

Comment:
Do we have an issue for this?  If not, recommend creating one and linking it here.  If so, recommend just linking it here.

Reply:
I don't think fill is getting used anywhere in pysat core. Looked through Instrument, meta, load_netcdf. I believe the undocumented rule within pysat is fill data should be NaN. I'll make an issue and note.

I do have a memory of fill being connected to file reading and writing but so far looking through the code I don't see it. Unsure if those are in penumbra packages now or what.